docs/topics/items_with_same_priority.md

```diff
@@ -10,7 +10,7 @@ The priority is equivalent.
     This approach may feel uncomfortable since CVSS gives the appearance of a finer grained priority.
     CVSS appears to say, 
     > Not just 4.0 to 6.9 is ‘medium’ severity, but 4.6 is more severe than 4.5.
-    However, as discussed {== previously (see page 4) ==}, CVSS is designed to be accurate only within +/- 0.5,
+    However, CVSS is designed to be accurate only within +/- 0.5,
```

Comment:
Did you intend for this change to be part of this PR? 

If not, I have a hunch that the problem is that you're working from the main branch in your fork rather than creating separate branches for separate changes.